src/components/input/toggle.tsx

```diff
@@ -0,0 +1,56 @@
+import { ElementSize } from "@structs/size";
+import React, { useRef } from "react";
+import styles from "./toggle.module.scss";
+
+function Toggle({
+  label,
+  name,
+  disabled,
+  size = "md",
+  value,
+  onChange,
+}: Props) {
+  const text = label ?? name;
+  const input = useRef<HTMLInputElement>(null);
+
+  return (
+    <fieldset
+      className={styles.main}
+      data-disabled={disabled}
```

Comment:
`fieldset` [supports the `disabled` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#attr-disabled). Any reason to use a custom `data-*` one here?